.github/workflows/daily-integration-check.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     env:
       SUPERSTAQ_API_KEY : ${{ secrets.SUPERSTAQ_API_KEY }}
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
5 min instead of 15?

Reply:
5 minutes for the individual Hilbert test. 15 for the whole suite.